SPECS/cmake/cmake.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
   "Signatures": {
     "macros.cmake": "1b5f1916a2118b932b217c4c5c4d52e723b1cf4c9587fe7f8fa45b41abfa7c60",
-    "cmake-3.29.6.tar.gz": "1391313003b83d48e2ab115a8b525a557f78d8c1544618b48d1d90184a10f0af"
+    "cmake-3.30.3.tar.gz": "6d5de15b6715091df7f5441007425264bdd477809f80333fdf95f846aaff88e4"
```

Comment:
Since this changes the source tarball, you'll also need to update `cgmanifest.json` so it point to the new URL.